### ISSUE TITLE
Add instructions for installing inside GitHub actions

### ISSUE THREAD

**Issue by cubranic** [pyFFTW/pyFFTW#309]
I have a package that has pyFFTW as a dependency and I don't have any problems installing it except in Github actions. There, it fails with the same "unknown file type '.pyx'" error from #252 . The solution was to have the following lines in the installation workflow:
```
python -m pip install --upgrade pip wheel setuptools cython
sudo apt install -y libfftw3-dev
```
Is this the right way to go about it? Either way, it took a long time to figure it out by trial and error (or rather, commit-push-see output listing), and it would have been really useful to have that mentioned in the Readme.

**Comment by hgomersall** [comment]
That looks reasonable to me. I guess the github actions couldn't find a pre-compiled wheel? Certainly, if you build from scratch then you'll need to have `libfftw3-dev` installed. For building, `FFTW` _is_ listed as a requirement. It's not clear to me why github actions needed to try to build from scratch. What is it looking for at PyPI?

**Comment by cubranic** [comment]
Without installing anything extra and just having `pip install pyfftw`, I see the following in the action’s output:
```
Collecting pyfftw~=0.12.0
52
 <https://github.com/chime-sps/ps-processes/runs/2104580876#step:4:52>  Downloading pyFFTW-0.12.0.tar.gz (137 kB)
53
 <https://github.com/chime-sps/ps-processes/runs/2104580876#step:4:53>
Using legacy 'setup.py install' for pyfftw, since package 'wheel' is not installed.
73
 <https://github.com/chime-sps/ps-processes/runs/2104580876#step:4:73>
```
Even after I installed `wheel` first, it still tried to build and failed with no fftw3.h header. Then I added that `apt install libfftw3-dev` and got the `pyx` error. Only after I also added cython did it start working, and now I get:
```
  Building wheel for pyfftw (setup.py): started
219
 <https://github.com/chime-sps/ps-processes/runs/2107033727#step:4:219>  Building wheel for pyfftw (setup.py): finished with status 'done'
220
 <https://github.com/chime-sps/ps-processes/runs/2107033727#step:4:220>  Created wheel for pyfftw: filename=pyFFTW-0.12.0-cp39-cp39-linux_x86_64.whl size=530907 sha256=2c6b7fb685e296663a17563045abfe3c36a458d23fe4575de0735ecbefe73715
221
 <https://github.com/chime-sps/ps-processes/runs/2107033727#step:4:221>  Stored in directory: /home/runner/.cache/pip/wheels/95/bc/73/66e2e5ff5feb06a4ab4a8adbc1b4a64ebd8713ff9da83f05d0
222
 <https://github.com/chime-sps/ps-processes/runs/2107033727#step:4:222>
```

**Comment by hgomersall** [comment]
Ah, it's because you're using python 3.9, so the source version isn't up on PyPI. This problem should go away.

**Comment by grlee77** [comment]
should be addressed by the recent 0.13.0 release
